Healthy/Classes/Modules/SearchReceipes /Cell/SearchCollectionViewCell.swift

```diff
@@ -0,0 +1,62 @@
+import UIKit
+
+class SearchCollectionViewCell: UICollectionViewCell {
+
+    // MARK: Properties
+
+    static let reusableIdentifire = "SearchCollectionViewCell"
+
+    // MARK: Outlets
+
+    @IBOutlet weak var backImage: UIImageView!
+    @IBOutlet weak var ratingNumber: UILabel!
+    @IBOutlet weak var recipyName: UILabel!
+    @IBOutlet weak var chefName: UILabel!
+    @IBOutlet weak var ratingView: UIView!
+
+    // MARK: Set up layout
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+        configureUI()
+    }
```

Comment:
```suggestion
    override func awakeFromNib() {
        super.awakeFromNib()
        
        configureLayout()
    }
```

`layoutSubviews` will be called every time this cell needs layout.

Reply:
Only rating view corner radius and gradient updates to be added to `layoutSubviews`

Reply:
This is tricky since you need to replace the existing gradient, let's see how you will handle it.